doc/source/environment_variables.rst

```diff
@@ -69,7 +102,37 @@ Environment Variables
      - Interfaces to listen on.
    * - EPICS_CAS_IGNORE_ADDR_LIST
      - ''
-     - Interfaces to igonre.
+     - Interfaces to ignore.
+   * - CAPROTO_SERVER_HIGH_LOAD_TIMEOUT
+     - 0.01
+     - Tuning this parameters will affect the server's performance under high
+       load. If the queue of subscriptions to has a new update ready within
+       this timeout, we consider ourselves under high load and trade accept
+       some latency for some efficiency.
+   * - CAPROTO_SERVER_HIGH_LOAD_EVENT_TIME_THRESHOLD
+     - 0.1
+     - If events are toggled by the client, subscriptions values get garbage-
+       collected.  It's not a high load situation.  Let's warn only if we're
+       relatively sure that it wasn't due to recent event toggling.  You
+       probably shouldn't need to change this.
+   * - CAPROTO_SERVER_HIGH_LOAD_WARN_LATENCY_SEC
+     - 0.03
+     - Warn the user if packets are delayed by more than this amount: 30ms.
+       Set to 0 to disable the warning entirely.
+   * - CAPROTO_SERVER_SUB_BATCH_THRESH
+     - 2 ** 16
+     - When a batch of subscription updates has this many bytes or more, send
+       it.
+   * - CAPROTO_SERVER_MAX_LATENCY
+     - 1
+     - Tune this to change the max time between packets. If it's too high, the
```

Comment:
units?  I assume seconds?

Reply:
Good point - I included units in all relevant ones in the docs and switched up env var names to reflect that